packages/frontend/src/components/common/Table/ScrollableTable/TableHeader.tsx

```diff
@@ -95,7 +95,8 @@ const TableHeader = () => {
                                                 </ThLabelContainer>
 
                                                 <ThActionsContainer>
-                                                    {meta?.item &&
+                                                    {(meta?.item ||
+                                                        meta?.isInvalidItem) &&
```

Comment:
What if we removed this condition completely and let the HeaderContextMenu decide when/what  to render ?

Reply:
Yeah, that makes sense. I'll update